src/ui/SubredditSelectorView.tsx

```diff
@@ -15,56 +15,40 @@ export default class SubredditSelectorView
     ChangeListeners<common.SelectionIndexChangeEventTarget, 'index'>;
 
   private _items: string[] = [];
```

Comment:
You should be able to just replace this with a public @property "items" that the ListView binds to.